tests/python/utils.py

```diff
@@ -126,3 +128,84 @@ def atexit_serde_check():
     fc = FusionCache.get()
     # Clear FusionCache because the tests expect a new fusion to be generated.
     FusionCache.reset()
+
+
+def serde_check(test_fn: Callable):
```

Comment:
Rename to something like `nvfusertest_serde_check`. This function has specific handling for `NVFuserTest.exec_nvfuser`